src/__tests__/ca/verify.test.ts

```diff
@@ -24,7 +24,9 @@ describe('verifySigningCertificate', () => {
   const bundleJSON = translateLegacyBundleJSON(
     bundles.dsse.valid.withSigningCert
   );
-  const bundle = sigstore.Bundle.fromJSON(bundleJSON);
+  const bundle = sigstore.Bundle.fromJSON(
+    bundleJSON
+  ) as sigstore.BundleWithCertificateChain;
```

Comment:
Since this is just test data and I know exactly what I've loaded, I'm just going to cast it to the type I need for the tests.